pkg/scheduler/framework/plugins/podtopologyspread/common.go

```diff
@@ -82,3 +82,17 @@ func filterTopologySpreadConstraints(constraints []v1.TopologySpreadConstraint,
 	}
 	return result, nil
 }
+
+func countPodsMatchSelector(pods []*v1.Pod, selector labels.Selector, ns string) int {
```

Comment:
nit: change int to int64 so we don't need to add explicit `int64(num)` conversion later.

Reply:
I plan to reuse this in Filter, which uses `int32` for whatever reason. `int` seems more portable.